tech.json

```diff
@@ -917,7 +917,7 @@
                 "Performing a canWaterShineCharge in a shorter runway by using a stutter that is greatly extended by the water slowdown.",
                 "Perform a stutter shortly before the water entrance, typically a door transition, by briefly releasing and then repressing forward repressing forward before, but as close to, the transition as possible while continuing the dash.",
                 "This carries the stutter into the water where Samus will run in place while building the shinecharge, Allowing Samus to enter the water with little momentum.",
-                "After entering the water, dash should be held the entire time."
+                "After entering the water, dash should be held the entire time. It is possible to do this between two adjacent submerged rooms with Gravity Suit, by removing the suit just before the transition."
```

Comment:
More generally, you would do the stutter before the door transition.

Turning off Gravity the frame before entering a door transition shouldn't be encouraged.  It raises trick difficulty a notch, and can give you the black screen which may be confusing.